src/main/java/org/openhab/binding/zigbee/converter/ZigBeeConverterColorColor.java

```diff
@@ -61,23 +76,66 @@ public void initializeConverter() {
             return;
         }
 
+        clusterOnOff = (ZclOnOffCluster) endpoint.getInputCluster(ZclOnOffCluster.CLUSTER_ID);
+        if (clusterOnOff == null) {
+            logger.error("Error opening device on/off controls {}", endpoint.getIeeeAddress());
+            return;
+        }
+
+        try {
+            if(!clusterColorControl.discoverAttributes(false).get()) {
+                logger.warn("{}: Cannot determine whether device endpoint supports RGB color. Assuming it supports HUE/SAT", endpoint.getEndpointAddress());
```

Comment:
This probably should be a debug level.

Reply:
Here it is the discovery itself failing, so we are assuming the endpoint supports Hue/Sat when it may not. In my understanding discovery should not normally fail, so hence the "warn", but I can put it as debug.